File: C/program22.c
#include<stdio.h>
#include<stdbool.h>

int main()
{
    int iValue = 0;
    bool bRet = false;

    printf("Enter number\n ");
    scanf("%d",&iValue);

    bRet = CheckDivisible(iValue);

    if (bRet == true)
    {
        printf("%d is divible by 5",iValue);
    }
    else
    {
        printf("%d number is not divible ",iValue);
    }

   return 0; 
}

bool CheckDivisible(int iNo) 
{
    bool bResult = false;
    if((iNo % 5 ) == 0)
    {
       bResult = true;
    }
    else
    {
        bResult = false;
    }

    return bResult;
}

File: Java/program22.java
import java.util.*;

class Divisible
{
    public boolean CheckDivisible(int iNo)
    {
        boolean bResult = false;
        if((iNo % 5) == 0)
        {
            bResult = true;
        }
        else
        {
           bResult = false;
        }
        
        return bResult;
    } 
}

class program22
{
    public static void main(String A[])
    {
        int iValue = 0;
        boolean bRet = false;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number :");
        iValue = sobj.nextInt();

        Divisible dobj = new Divisible();
        bRet = dobj.CheckDivisible(iValue);

        if(bRet == true)
        {
            System.out.println(iValue +" is divisible by 5");

        }
        else
        {
            System.out.println(iValue +" is not divisible by 5");
        }

               

        
    }
}